holmes/config.py

```
@@ -242,6 +148,22 @@ def model_post_init(self, __context: Any) -> None:
                 self._holmes_info.latest_version,
             )
 
+    @classmethod
+    def load_from_file(cls, config_file: Optional[Path], **kwargs) -> "Config":
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)